src/test/resources/executeafm/afm/popMeasureDefinition.json

```diff
@@ -5,4 +5,4 @@
       "uri": "/gdc/md/projectId/obj/1"
     }
   }
-}
```

Comment:
its a cleanup?

Reply:
Each file should end with a line break. This one didn't have it for some reason.